tests/integration/integration-tests.yml

```diff
@@ -1906,6 +1906,27 @@
           - test_five['device_type'] == None
           - test_five['msg'] == "device_type test-device-type already absent"
 
+    - name: "DEVICE_TYPE 6: Without Slug"
+      netbox_device_type:
+        netbox_url: http://localhost:32768
+        netbox_token: 0123456789abcdef0123456789abcdef01234567
+        data:
+          model: ws-test-3850
```

Comment:
what about to name model `WS Test 3850` to make sure that slugifying works fine?

Reply:
Yes that is a very good catch. I will update that test this weekend